generators/entity/templates/src/test/java/package/web/rest/_EntityResourceIntTest.java

```diff
@@ -195,7 +195,8 @@ public void setup() {
 
     @Before
     public void initTest() {<% if (databaseType == 'mongodb' || databaseType == 'cassandra') { %>
-        <%= entityInstance %>Repository.deleteAll();<% } %>
+        <%= entityInstance %>Repository.deleteAll();<% } if (searchEngine == 'elasticsearch') { %>
+        <%= entityInstance %>SearchRepository.deleteAll();<% } %>
```

Comment:
Ill look at them today
On 17 Mar 2016 03:38, "Gerald Humphries" <notifications@github.com> wrote:

> In
> generators/entity/templates/src/test/java/package/web/rest/_EntityResourceIntTest.java
> <https://github.com/jhipster/generator-jhipster/pull/3168#discussion_r56400756>
> :
>
> > @@ -195,7 +195,8 @@ public void setup() {
> >
> >      @Before
> >      public void initTest() {<% if (databaseType == 'mongodb' || databaseType == 'cassandra') { %>
> > -        <%= entityInstance %>Repository.deleteAll();<% } %>
> > +        <%= entityInstance %>Repository.deleteAll();<% } if (searchEngine == 'elasticsearch') { %>
> > +        <%= entityInstance %>SearchRepository.deleteAll();<% } %>
>
> They won't change during the tests, but the ES indexes persist (they're
> stored on the hard drive) so the mappings may change between test sessions.
> That's why I was trying to find a solution to delete all indexes before any
> test is run.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/jhipster/generator-jhipster/pull/3168/files/5f9abafc5f318de35a7b91a1c4196c26afaa7446#r56400756>
>